torchvision/datasets/voc.py

```diff
@@ -4,8 +4,8 @@
 from .vision import VisionDataset
 import xml.etree.ElementTree as ET
 from PIL import Image
-from typing import Any, Callable, Dict, List, Optional, Tuple, Union
-from .utils import download_url, check_integrity, verify_str_arg
+from typing import Any, Callable, Dict, List, Optional, Tuple
```

Comment:
I believe List is not used either?
```suggestion
from typing import Any, Callable, Dict, Optional, Tuple
```